### PR TITLE
messageUpdate triggering on false positives

### PR DESCRIPTION
**Pull request by bagladivyang03**
# Pull Request

## Description:
[BUG] - messageUpdate triggering on false positives #528

## Related Issue:

Closes #XXXXX

## Scope:

Did you remember to update the `package.json` version number?

- [ ] Major Version Update: X.\_.\_ (for complete code refactors, or large PRs that adjust most of the codebase)
- [ ] Minor Version Update: \_.X.\_ (for the addition of new commands)
- [x] Patch Version Update: \_.\_.X (for bug fixes _in the code_.)
- [ ] No Version Update: \_.\_.\_ (no version update for additions to tests, documentation, or anything that isn't end-user facing.)

## Documentation:

For _any_ version updates, please verify if the [documentation page](https://beccalyria.nhcarrigan.com) needs an update. If it does, please [create an issue there](https://github.com/BeccaLyria/discord-documentation/issues/new?assignees=nhcarrigan&labels=%F0%9F%9A%A6+status%3A+awaiting+triage&template=update.md&title=%5BUPDATE%5D) to ensure it is not forgotten.

- [x] My contribution does NOT require a documentation update.
- [x] My contribution DOES require a documentation update.

### CODE REVIEW

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
A couple of quick things:

- You'll need to fix the linting issues for the CI to pass.
- After you've fixed the linting issues, you'll need to bump the version. You can do this with `npm version patch` and a commit will be generated which you'll push up to your branch.

**Comment by bagladivyang03** [comment]
#528

**Comment by bagladivyang03** [comment]
> A couple of quick things:
> 
> * You'll need to fix the linting issues for the CI to pass.
> * After you've fixed the linting issues, you'll need to bump the version. You can do this with `npm version patch` and a commit will be generated which you'll push up to your branch.

Could u pls tell me in brief how to do this I m a beginner. So it would be great if you could help me!

**Comment by nhcarrigan** [comment]
Are you working on this directly through GitHub? If so, I recommend cloning it locally so you can use the included tooling to fix the linting issues.

File: src/events/onMessageUpdate.ts [hunk 1]
```diff
@@ -21,6 +21,10 @@ async function onMessageUpdate(
     // Get the author and the current server from the new message.
     const { author, guild } = newMessage;
 
+    // validation added if newMessage and oldMessage are same
+    if (      oldMessage.content &&      newMessage.content &&      oldMessage.content === newMessage.content    ) {
+      return;
+    }
```
Comment (nhcarrigan) [inline src/events/onMessageUpdate.ts:27]:
```suggestion
    // validation added if newMessage and oldMessage are same
    if (
      oldMessage.content &&
      newMessage.content &&
      oldMessage.content === newMessage.content
    ) {
      return;
    }
```

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
Added a suggestion you can commit through GitHub to resolve the linting issue.

**Review by nhcarrigan — APPROVED** [review]
Looks good to me. Tested locally, resolves the false positives.

Will merge when the CI passes - I went ahead and pushed up the version commit too.

**Comment by nhcarrigan** [comment]
@all-contributors please add @bagladivyang03 for code

**Comment by bagladivyang03** [comment]
@nhcarrigan Thanks for the help.

**Comment by nhcarrigan** [comment]
@all-contributors please add @bagladivyang03 for code

**Comment by nhcarrigan** [comment]
Congrats on your first pull request (PR).

Thank you for your contribution to the page. We are happy to accept these changes and look forward to future contributions.

**Comment by nhcarrigan** [comment]
Try this again...

@all-contributors please add @bagladivyang03 for code

**Comment by nhcarrigan** [comment]
Try this again...

@all-contributors please add @bagladivyang03 for code

**Pull request merged**